### ISSUE TITLE
Auto Accept feature causes panic?

### ISSUE THREAD

**Issue by GermanCoding** [syncthing/syncthing#5165]
### Version Information

Syncthing Version: v0.14.49
OS Version: Debian Stretch (64bit)

### Background

I've a server which runs syncthing and mirrors some of my important data (+ performs backups ) to other computers. This server recently died (hardware failure) and I decided to cleanly re-setup most systems, including syncthing on a new machine. That means new synthing ID, the system appears as new on my syncthing cluster (+ has no folders).

### Events of the crash
First I re-uploaded some important folders to the server, but left out the big ones (80+ GB of data). Today I tried uploading the big folders from my laptop. I had set my laptop to auto accept in the server and switched the folders in question to "share with [server]" on my laptop.

Afterwards syncthing disconnected the connection and after it was rebuild syncthing (EDIT: I meant the server, the receiver) immediatly died with a panic, but restarted automatically which then caused a loop of infinite panics whenever my laptop attempted a connection to the server.

### Logging
The log says:

`[K4D7W] 13:08:02 INFO: Adding folder "Uni" (u4tbq-v3wu5)`
`Panic at 2018-09-06T13:08:02+02:00`
`panic: cannot start already running folder u4tbq-v3wu5`

Interesting: This folder was already shared between laptop and server (it was one of the intial re-uploads), "all was well". Only after ticking additional folders in the share with window (a week later after uploading this folder) this seemed to cause issues. The folder was fully in sync on all machines before this event.

Eventually I was able to solve the problem by switchting auto accept off in the server. After this the server seemed to "understand" that it already had this folder, as it is no longer mentioned in the last-non panic log. I was asked to manually accept each folder which I did, then everything worked.

Full panic log (censored minor personal data) attached, I've about a dozen others of them (due to infinetly repeating panic) but they all appear to tell the same error.

[panic-20180906-130552.log](https://github.com/syncthing/syncthing/files/2356866/panic-20180906-130552.log)

The last log, after switchting off auto accept which does not include panics is not attached because it does not seem to contain anything of interest and it would take a while to sweep out some private data. If it is really needed let me know, I've secured the log into a file.

**Comment by imsodin** [comment]
Duplicate of #5063
